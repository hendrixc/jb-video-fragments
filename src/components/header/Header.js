import React, { Component } from 'react';
import '@material/toolbar/dist/mdc.toolbar.css';

import {
  Toolbar,
  ToolbarRow,
  ToolbarTitle,
} from '@rmwc/toolbar';

class Header extends Component {
  render() {
    return (
      <div className="Header">
        <Toolbar>
          <ToolbarRow>
            <ToolbarTitle>Toolbar</ToolbarTitle>
          </ToolbarRow>
        </Toolbar>
      </div >
    );
  }
}

export default Header;
