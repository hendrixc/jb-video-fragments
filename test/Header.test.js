import React from 'react';
import renderer from 'react-test-renderer';
import Header from '../src/components/header/Header';

it('renders an home component into the Header as snapshot', () => {
  const TextInputComponent = renderer.create(<Header />).toJSON();
  expect(TextInputComponent).toMatchSnapshot();
});
