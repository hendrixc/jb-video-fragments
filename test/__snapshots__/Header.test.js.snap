// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders an home component into the Header as snapshot 1`] = `
<div
  className="Header"
>
  <header
    className="mdc-top-app-bar"
    style={Object {}}
  >
    <div
      className="mdc-top-app-bar__row"
    >
      <section
        className="mdc-top-app-bar__section"
      >
        <i
          className="rmwc-icon material-icons mdc-top-app-bar__navigation-icon"
          data-mdc-ripple-is-unbounded={true}
          onBlur={[Function]}
          onFocus={[Function]}
          onKeyDown={[Function]}
          onKeyUp={[Function]}
          onMouseDown={[Function]}
          onMouseUp={[Function]}
          onTouchEnd={[Function]}
          onTouchStart={[Function]}
          style={Object {}}
        >
          menu
        </i>
        <div
          className="mdc-top-app-bar__title"
        >
          JB Video Fragments Tube
        </div>
      </section>
      <section
        className="mdc-top-app-bar__section mdc-top-app-bar__section--align-end"
      >
        <i
          className="rmwc-icon material-icons mdc-top-app-bar__navigation-icon"
          data-mdc-ripple-is-unbounded={true}
          onBlur={[Function]}
          onFocus={[Function]}
          onKeyDown={[Function]}
          onKeyUp={[Function]}
          onMouseDown={[Function]}
          onMouseUp={[Function]}
          onTouchEnd={[Function]}
          onTouchStart={[Function]}
          style={Object {}}
        >
          favorite
        </i>
      </section>
    </div>
  </header>
</div>
`;
